.config/rollup.config.mjs

```diff
@@ -128,7 +84,7 @@ function configCore( input, filename, plugins ){
 
 	var output = {
 		name: 'TomSelect',
-		file: `build/js/${filename}`,
+		file: `dist/js/${filename}`,
```

Comment:
AFAICT the `build`/`dist` distinction wasn't useful. I opted to just directly construct everything in `dist`, which simplified things a bit.